src/main/java/uk/gov/companieshouse/company_appointments/interceptor/AuthenticationHelperImpl.java

```diff
@@ -66,7 +66,7 @@ public String getAuthorisedUser(HttpServletRequest request) {
     public String getAuthorisedUserEmail(HttpServletRequest request) {
         final String authorisedUser = getAuthorisedUser(request);
 
-        if (authorisedUser == null || authorisedUser.trim().length() == 0) {
+        if (authorisedUser == null || authorisedUser.trim().isEmpty()) {
```

Comment:
Rather than have a null check and a empty check seperate, we could use

import com.google.common.base.Strings; 
.isNullOrEmpty()

Isn't a necessary change, just a thought :)

Reply:
will leave as is since .trim() is used in many instances of this logic